### PR TITLE
INCUBATOR-202 Proposal for enhancing graduation steps description

### PR DESCRIPTION
**Pull request by ebarboni**
An attempt to enhance the graduation steps as we (Apache NetBeans) encounter 404 error on the Service Desk Link.
Also trying to split the PMC chair and PMC "todolist".
Fixed some links
Hope it helps

### CODE REVIEW

File: pages/guides/transferring.ad [hunk 1]
```diff
@@ -38,69 +39,80 @@ themselves.
...
+
 When graduating to a new project, the process is more
 complex. Creating a new project requires a
-link:http://www.apache.org/foundation/board/calendar.html[resolution]
+link:graduation.html#preparing_a_charter[resolution]
 to be passed by the link:http://www.apache.org/foundation/board/[Board].
-Usually once this happens, the secretary will
+You may look at older minutes from the link:http://www.apache.org/foundation/board/calendar.html[calendar] and check for "Establish ProjectName"
+
+The passed will appoint Project Managnement Committees:
```
Comment (ottlinger) [inline pages/guides/transferring.ad]: typo: Managnement -> Management

I personally do not understand the sentence:
"The passed will appoint PMC" - is it supposed to be "If passed the newly formed PMC will:"?

File: pages/guides/transferring.ad [hunk 2]
```diff
@@ -38,69 +39,80 @@ themselves.
...
 hours has passed since the Board meeting, it may be
 worth pinging the board to request confirmation.
 
-The link:#tlp-resolution[resolution] will appoint a Chair for the new
-project. The Chair will also be appointed an
-link:http://www.apache.org/foundation/[Officer]
-of the Apache Software Foundation. This allows them
-access to official resources of the foundation as well
-as granting power to act on behalf of Apache.
+- the initial member of Project Managnement Committees aka PMC member.
```
Comment (ottlinger) [inline pages/guides/transferring.ad]: typo: Managnement -> Management

File: pages/guides/transferring.ad (hunk 1 above)
Comment (ebarboni) [reply pages/guides/transferring.ad]: can this be said:
The board (if the resolution is passed) will appoint Project Managnement Committees composed of

Comment (ShaneCurcuru) [reply pages/guides/transferring.ad]: To be more precise:
```suggestion
The Board votes on graduation resolutions at their monthly meeting.  If the Board votes to pass the resolution, that then appoints the listed Project Management Committee including:
```
Resolutions are formal acts of the ASF, which the board votes to pass (or occasionally defer to a later meeting).  Once the board votes, the actions written in the resolution are formally done (like appointing PMC members and the VP officer).

File: pages/guides/transferring.ad (hunk 2 above)
Comment (ShaneCurcuru) [reply pages/guides/transferring.ad]: Addition (should have been here originally):

> s/act on behalf of Apache./act on behalf of Apache within the scope of their project's activities./

PMC Chairs are only empowered to speak on behalf of their project and to accomplish the duties for their specific project.

Comment (ebarboni) [reply pages/guides/transferring.ad]: addition changes done in commit 2d1c717.

Comment (ottlinger) [reply pages/guides/transferring.ad]: @ebarboni the typo is still there. Not sure if @ShaneCurcuru 's additions should be added as well.

Comment (ebarboni) [reply pages/guides/transferring.ad]: sorry it's on the next commit.

Comment (ottlinger) [reply pages/guides/transferring.ad]: @ebarboni I'd say just commit the changes, should not be a problem.

**Comment by ottlinger** [comment]
Thanks for your contribution :+1:

**Comment by ebarboni** [comment]
you're welcome. 
Sorry I was on another bad typo management typo

**Pull request merged**